Source/OxyPlot/Rendering/OxyAnnulus.cs

```diff
@@ -0,0 +1,207 @@
+        /// Gets the inner left of the annulus.
+        /// </summary>
+        /// <value>The most left inner point.</value>
+        public ScreenPoint InnerLeft => new ScreenPoint(this.Center.X - this.InnerRadius, this.Center.Y);
+
+        /// <summary>
+        /// Gets the inner right of the annulus.
+        /// </summary>
+        /// <value>The most right inner point.</value>
+        public ScreenPoint InnerRight => new ScreenPoint(this.Center.X + this.InnerRadius, this.Center.Y);
+
+        /// <summary>
+        /// Gets the outer top of the annulus.
+        /// </summary>
+        /// <value>The top outer point.</value>
+        public ScreenPoint OuterTop => new ScreenPoint(this.Center.X, this.Center.Y + this.OuterRadius);
+
+        /// <summary>
+        /// Gets the outer bottom of the annulus.
+        /// </summary>
+        /// <value>The bottom outer point.</value>
+        public ScreenPoint OuterBottom => new ScreenPoint(this.Center.X, this.Center.Y - this.OuterRadius);
+
+        /// <summary>
+        /// Gets the outer left of the annulus.
+        /// </summary>
+        /// <value>The most left outer point.</value>
+        public ScreenPoint OuterLeft => new ScreenPoint(this.Center.X - this.OuterRadius, this.Center.Y);
+
+        /// <summary>
+        /// Gets the outer right of the annulus.
+        /// </summary>
+        /// <value>The most right outer point.</value>
+        public ScreenPoint OuterRight => new ScreenPoint(this.Center.X + this.OuterRadius, this.Center.Y);
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="OxyAnnulus"/> structure that has a specified center a inner and outer radius
+        /// </summary>
+        /// <param name="center">The center location of the annulus</param>
+        /// <param name="innerRadius">The inner radius of the annulus</param>
+        /// <param name="outerRadius">The outer radius of the annulus</param>
+        /// <exception cref="System.ArgumentOutOfRangeException">r1;The inner radius should not be negative.</exception>
+        /// <exception cref="System.ArgumentOutOfRangeException">r2;The outer radius should not be equal or smaller as the inner radius.</exception>
+        public OxyAnnulus(ScreenPoint center, double innerRadius, double outerRadius)
+        {
+            if (innerRadius < 0)
+                throw new ArgumentOutOfRangeException(nameof(innerRadius), "The inner radius should not be negative.");
+            if (outerRadius <= innerRadius)
+                throw new ArgumentOutOfRangeException(nameof(outerRadius), "The outer radius should not be equal or smaller as the inner radius.");
+
+            this.Center = new ScreenPoint(center.X, center.Y);
+            this.InnerRadius = innerRadius;
+            this.OuterRadius = outerRadius;
+        }
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="OxyAnnulus"/> structure that has a specified center a inner and outer radius
```

Comment:
"has a specified center a inner and outer radius" -> "has the specified center, inner radius, and outer radius"